### DISCUSSION TITLE
Organization email?

### DISCUSSION THREAD

**Discussion by ferdnyc** [andyholmes/gnome-shell-extension-gsconnect#1522] [General]
Just now I experimentally tried emailing GSConnect@github.com to see if it went anywhere, and this was the response:

![image](https://user-images.githubusercontent.com/538020/203116928-c1c89167-5044-4473-a58e-1b0e42de1ae7.png)

Which sort of seems to maybe-imply that the address _could_ go somewhere, if the organization settings were changed. (Assuming the organization that message refers to is https://github.com/GSConnect/, and not GitHub itself.)

So the question is, should the organization's email settings (if any) be changed? And if so, what are the options?

(I'm thinking mostly for license-contact purposes, since tools like REUSE generally want an email address to associate with the entity declared as license holder for a project.)

**Comment by daniellandau** [comment]
Didn't find anything that seemed relevant in the settings.

**Reply by ferdnyc** [reply]
Mm, all of the documentation [in the first docs result for an "organization email" search](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) is about how _users_ can configure their own email notifications for organization-related activity. It doesn't sound like an "organization email address" is really a thing, at least not a thing provided by GitHub.

**Comment by ferdnyc** [comment]
Closing this as investigated and dismissed.
